src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -403,6 +403,9 @@ public enum AddressFamily
         Irda = 26,
         NetworkDesigners = 28,
         Max = 29,
+        Netlink = 30,
```

Comment:
How can we be adding enum values after the "Max" enum value?  I guess "Max" isn't really "Max" anymore??

Reply:
no it is not. We concluded that was carried from old C code.  And documentation never established any special meaning of it.